cro-clib/android_mac.md

```diff
@@ -0,0 +1,95 @@
+## install android for mac
+1. install android studio
+2. install ndk, sdk(api 26)
+3. locations as below  
+```
+sdk:  $HOME/Library/Android/sdk
+ndK:  $HOME/Library/Android/sdk/ndk/21.0.6113669
+```
+
+## modify path 
+to make it compatible with linux script  
+```
+ln -sf $HOME/Library/Android $HOME/Android
+```
+
+## make rust toolchain folder  
+cross compiler will be copied here for rust  
+```
+mkdir -p $HOME/rust_android/NDK
+```
+
+## check folders
+now folders are like below 
+```
+ANDROID_HOME: l$HOME/Android/Sdk
+NDK_HOME: $ANDROID_HOME/ndk/21.0.6113669
+RUST_ANDROID: $HOME/rust_android/NDK
+```
+
+
+## install openssl
+install openssl   
+```
+brew install openssl@1.1
+```
+
+## go to folder
+```
+cd chain/cro-clib
+```
+
+
+## check crate-type
+1. vi Cargo.toml  in cro-clib folder
+2. modify crate-type as staticlib only  
+```
+[lib]
+name = "cro_clib"
+crate-type =["staticlib"]
+```
```

Comment:
can it be without this step if Cargo.toml is changed?

Reply:
i'll remove it